.github/workflows/package-gate.yml

```
@@ -0,0 +1,46 @@
+      - main
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  package-gate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      NUGET_PACKAGES: ${{ runner.temp }}/nuget-packages
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)